knowledge-graph-identification/java/src/main/java/org/linqs/psl/examples/kgi/Run.java

```diff
@@ -97,37 +97,40 @@ private void definePredicates() {
     private void defineRules() {
         log.info("Defining model rules");
 
-           model.addRules("0025 : VALCAT(B, C) & SAMEENTITY(A, B) & CAT(A, C) -> CAT(B, C) ^2"+
-            "0025: VALREL(B, Z, R) & SAMEENTITY(A, B) & REL(A, Z, R) -> REL(B, Z, R) ^2"+
-            "0025: VALREL(Z, B, R) & SAMEENTITY(A, B) & REL(Z, A, R) -> REL(Z, B, R) ^2"+
-            "0100: VALCAT(A, D) & SUB(C, D) & CAT(A, C) -> CAT(A, D) ^2"+
-            "0100: VALREL(A, B, S) & RSUB(R, S) & REL(A, B, R) -> REL(A, B, S) ^2"+
-            "0100: VALCAT(A, D) & MUT(C, D) & CAT(A, C) -> !CAT(A, D) ^2"+
-            "0100: VALREL(A, B, S) & RMUT(R, S) & REL(A, B, R) -> !REL(A, B, S) ^2"+
-            "0100: VALREL(B, A, S) & INV(R, S) & REL(A, B, R) -> REL(B, A, S) ^2"+
-            "0100: VALCAT(A, C) & DOMAIN(R, C) & REL(A, B, R) -> CAT(A, C) ^2"+
-            "0100: VALCAT(B, C) & RANGE2(R, C) & REL(A, B, R) -> CAT(B, C) ^2"+
-            "0001: VALCAT(A, C) & CANDCAT_GENERAL(A, C) -> CAT(A, C) ^2"+
-            "0001: VALREL(A, B, R) & CANDREL_GENERAL(A, B, R) -> REL(A, B, R) ^2"+
-            "0001: VALCAT(A, C) & CANDCAT_CBL(A, C) -> CAT(A, C) ^2"+
-            "0001: VALREL(A, B, R) & CANDREL_CBL(A, B, R) -> REL(A, B, R) ^2"+
-            "0001: VALCAT(A, C) & CANDCAT_CMC(A, C) -> CAT(A, C) ^2"+
-            "0001: VALCAT(A, C) & CANDCAT_CPL(A, C) -> CAT(A, C) ^2"+
-            "0001: VALREL(A, B, R) & CANDREL_CPL(A, B, R) -> REL(A, B, R) ^2"+
-            "0001: VALCAT(A, C) & CANDCAT_MORPH(A, C) -> CAT(A, C) ^2"+
-            "0001: VALCAT(A, C) & CANDCAT_SEAL(A, C) -> CAT(A, C) ^2"+
-            "0001: VALREL(A, B, R) & CANDREL_SEAL(A, B, R) -> REL(A, B, R) ^2"+
-            "0001: VALCAT(A, C) & PROMCAT_GENERAL(A, C) -> CAT(A, C) ^2"+
+           model.addRules(
+
+            "0025 : VALCAT(B, C) & SAMEENTITY(A, B) & CAT(A, C) -> CAT(B, C) ^2" +
```

Comment:
Spacing here is weird (more so that the other multiline rules).
The rules has an extra space character before the quotes.
This string is an argument to addRules() and should be indented.